rest/model/project.go

```diff
@@ -551,6 +551,9 @@ type APIGitHubDynamicTokenPermissionGroup struct {
 	Name *string `json:"name"`
 	// Permissions for the GitHub permission group.
 	Permissions map[string]string `json:"permissions"`
+	// AllPermissions is true if the group has all permissions.
+	// This would appear as an empty 'Permissions' map.
+	AllPermissions *bool `json:"all_permissions"`
```

Comment:
Can you please elaborate more on "This would appear as an empty 'Permissions' map." here? Empty permissions map can also be no permissions so I think it can use more elaboration.

Reply:
ah never mind, I see we have this